### ISSUE TITLE
Prescription page - Unauthenticated users main content updates

### ISSUE THREAD

**Issue by ambikaroos** [department-of-veterans-affairs/va.gov-team#5741]
**User Story**
As a user who is browsing the Prescription page on VA.gov without authenticating (logging in), I want the content on the page (above and below the alert box) to be neutral between MHV and Cerner so that I am not confused about where my patient portal information is housed.

**Acceptance Criteria**
1. Content on Prescriptions page (above and below the alert box) for unauthenticated users does not contain any references to MHV or Cerner

**Tasks**
- [x] SSO-Login team: First pass at copy changes
- [x] VSP Content-IA team: Edit and validate
- [ ] SSO-Login team: Ensure frontend implementation

_Alert box content updates are housed in separate tickets_

**Comment by bridgethapner** [comment]
@peggygannon Wireframe is here: https://adhoc.invisionapp.com/share/DYVMI5OXEQN#/screens/404897006. I only changed 1 thing in the intro paragraph - it's outlined with a pink box & there is comments in the Invision Prototype explaining what I've changed and why. Since this page was 1 simple change I'm hoping it's set, but still appreciate any feedback on the entirety of the intro paragraph, or anything else down the page that you think could be improved.

**Comment by peggygannon** [comment]
Hi, @bridgethapner I do have content edits (below) for the content under "What if I have more questions" 

Also, I wanted to ask if we need to unbrand the VA Prescription and Refill Tracking tool? or is this still uppercase in the mock b/c Cerner doesn't support prescriptions at this time? 

These comments also appear in the mockup. thx! 


**What if I have more questions?** 
If you have questions about prescriptions and refills on MyHealtheVet, please go to the [Prescription refills FAQs]( https://www.myhealth.va.gov/mhv-portal-web/web/myhealthevet/faqs#PrescriptionRefill) on the My HealtheVet web portal. 
Or contact the My HealtheVet help desk at 877-327-0022 (TTY: 800-877-8339). We’re here Monday through Friday, 7:00 a.m. to 7:00 p.m. CT. 
You can also [contact us online] (https://www.myhealth.va.gov/mhv-portal-web/web/myhealthevet/contact-mhv

**Comment by bridgethapner** [comment]
Hi @peggygannon! We won't be linking to Cerner at launch, only to the phone number to refill Mann-Grandstaff prescriptions. I'm not sure when Cerner prescriptions will be available, but my POV is that we should go ahead and debrand now.

**Comment by bridgethapner** [comment]
Hi @peggygannon - We can discuss in our touch base tomorrow, but we heard last week that the launch date has been pushed back, now the Cerner prescription functionality _will_ be ready at the time of launch.

**Comment by bridgethapner** [comment]
I have [updated the UX and my first content pass](https://adhoc.invisionapp.com/share/DYVMI5OXEQN#/screens/404897006) in light of the fact that Cerner will offer online prescription manging at the time we launch. 

**Summary of edits (also in Invision):**
- removed "within My HealtheVet" from intro paragraph 
- removed "Removed ", sometimes called “Rx Refill” or “Rx Tracker,"" from the second section
- I implemented the "What if I have more questions?" edits sent by Peggy

I have left "Prescription Refill and Tracking tool" title case for now and continued to do so in the Auth version too. I noticed "Secure Messaging" is always title case too. Wasn't sure what to do, so left for now.
